docs/pull_request_template.md

```diff
@@ -7,3 +7,6 @@
 
 ## Visual Changes
 <!-- If the change results in visual changes, show a before and after -->
+
+## View Changes
+https://govuk-publishing-compon-pr-[PR-NUMBER].herokuapp.com/
```

Comment:
To avoid having a pull request with [PR-NUMBER] visible, we could wrap it in a comment? This would mean that the link can still be easily added if required, but won't show unless it's edited.

```suggestion
<!-- https://govuk-publishing-compon-pr-[PR-NUMBER].herokuapp.com/ -->
```